test/infrastructure/docker/exp/internal/controllers/dockermachinepool_controller.go

```diff
@@ -133,6 +135,19 @@ func (r *DockerMachinePoolReconciler) SetupWithManager(ctx context.Context, mgr
 		return err
 	}
 
+	log := ctrl.Log.WithName("remote").WithName("ClusterCacheTracker")
```

Comment:
let's create the tracker in the CAPD main.go (like in the top-level main.go file) and then pass it through via a `Tracker` field on the reconciler

Reply:
It's more consistent with other controllers and also has the advantage that we only have to create one tracker and can share them across reconcilers